airbyte-integrations/connectors/source-slack/source_slack/manifest.yaml

```diff
@@ -0,0 +1,307 @@
+version: 0.58.7
+type: DeclarativeSource
```

Comment:
Why we don't use the `latest` version possible?

Reply:
updated